### PR TITLE
docs: updates for empty dashboard state

### PR DESCRIPTION
**Pull request by imatwawana**
Updates related to dashboard empty state. Includes some updates related to the new navigation.

Do not merge until all related PRs are merged:
https://github.com/grafana/grafana/pull/65464

### CODE REVIEW

File: docs/sources/dashboards/build-dashboards/create-dashboard/index.md [hunk 1]
```diff
@@ -23,18 +23,19 @@ Dashboards and panels allow you to show your data in visual form. Each panel nee
 
 **To create a dashboard**:
 
-1. Sign in to Grafana, hover your cursor over **Dashboard**, and click **+ New Dashboard**.
-1. Click **Add a new panel**.
-1. In the first line of the **Query** tab, click the drop-down list and select a data source.
+1. Click **Dashboards** in the left-side menu.
+1. Click **New** and select **New Dashboard**.
```
Comment (polibb) [inline docs/sources/dashboards/build-dashboards/create-dashboard/index.md:27]: yeah, it's better as a separate point ☺️

**Review by polibb — APPROVED** [review]
very throughout! I love how consistent you made everything :)

File: docs/sources/dashboards/build-dashboards/create-dashboard/index.md [hunk 2]
```diff
@@ -23,18 +23,19 @@ Dashboards and panels allow you to show your data in visual form. Each panel nee
...
+1. On the empty dashboard, click **Add visualization**.
```
Comment (imatwawana) [inline docs/sources/dashboards/build-dashboards/create-dashboard/index.md:28]:
```suggestion
1. On the empty dashboard, click **+ Add visualization**.
```

File: docs/sources/dashboards/build-dashboards/create-dashboard/index.md [hunk 3]
```diff
@@ -64,28 +68,30 @@ To see an example of repeating rows, refer to [Dashboard with repeating rows](ht
 
 **To configure repeating rows:**
 
-1. On the dashboard home page, click **Add panel**.
+1. Click **Dashboards** in the left-side menu.
+1. Navigate to the dashboard you want to work on.
+1. At the top of the dashboard, click **Add** and select **Row** in the dropdown.
 
-1. On the **Add a panel** dialog box, click **Add a new row**.
+   If the dashboard is empty, you can just click the **Add row** option in the middle of the dashboard.
```
Comment (imatwawana) [inline docs/sources/dashboards/build-dashboards/create-dashboard/index.md:75]:
```suggestion
   If the dashboard is empty, you can just click the **+ Add row** button in the middle of the dashboard.
```

File: docs/sources/dashboards/build-dashboards/manage-library-panels/index.md [hunk 4]
```diff
@@ -40,12 +40,9 @@ Once created, you can modify the library panel using any dashboard on which it a
...
 
-1. Hover over the **Dashboards** option on the left menu, then select **New dashboard** from the drop-down options.
-
-   The **Add** panel dialog opens.
-   {{< figure src="/static/img/docs/library-panels/add-library-panel-8-0.png" class="docs-image--no-shadow" max-width= "900px" caption="Screenshot of the edit panel" >}}
-
-1. Click the **Add a panel from the panel library** option.
+1. Click **Dashboards** in the left-side menu.
+1. Click **New** and select **New Dashboard**.
+1. On the empty dashboard, click **Import panel**.
```
Comment (imatwawana) [inline docs/sources/dashboards/build-dashboards/manage-library-panels/index.md:45]:
```suggestion
1. On the empty dashboard, click **+ Import library panel**.
```

File: docs/sources/getting-started/build-first-dashboard.md [hunk 5]
```diff
@@ -40,15 +40,15 @@ To sign in to Grafana for the first time:
 To create your first dashboard:
 
 1. Click **Dashboards** in the left-side menu.
-1. On the Dashboards, click **New** and select **New Dashboard** from dropdown menu.
-1. On the dashboard, click **Add a new panel**.
+1. On the Dashboards page, click **New** and select **New Dashboard** from the dropdown menu.
+1. On the dashboard, click **Add visualization**.
```
Comment (imatwawana) [inline docs/sources/getting-started/build-first-dashboard.md:44]:
```suggestion
1. On the dashboard, click **+ Add visualization**.
```

File: docs/sources/tutorials/grafana-fundamentals/index.md [hunk 6]
```diff
@@ -215,16 +215,17 @@ Dashboards consist of _panels_, each representing a part of the story you want y
 
 Every panel consists of a _query_ and a _visualization_. The query defines _what_ data you want to display, whereas the visualization defines _how_ the data is displayed.
 
-1. In the sidebar, hover your cursor over the **Create** (plus sign) icon and then click **Dashboard**.
-1. Click **Add a new panel**.
-1. In the **Query editor** below the graph, enter the query from earlier and then press Shift + Enter:
+1. Click **Dashboard** in the left-side menu.
+1. On the Dashboards page, click **New** and select **New Dashboard** in the dropdown.
+1. Click **Add visualization**.
```
Comment (imatwawana) [inline docs/sources/tutorials/grafana-fundamentals/index.md:220]:
```suggestion
1. Click **+ Add visualization**.
```

File: docs/sources/dashboards/build-dashboards/create-dashboard/index.md [hunk 7]
```diff
@@ -48,9 +49,12 @@ Dashboards and panels allow you to show your data in visual form. Each panel nee
...
    - [Configure standard options]({{< relref "../../../panels-visualizations/configure-standard-options/" >}})
 
-1. Add a note to describe the visualization (or describe your changes) and then click **Save** in the upper-right corner of the page.
+1. When you've finished editing your panel, click **Save** in the top right corner.
+1. Enter a name for your dashboard and select a folder, if applicable.
+1. Click **Save**.
 
-   Notes can be helpful if you need to revert the dashboard to a previous version.
+<!-- this part doesn't make sense for a first time dashboard creation - this interaction only happens when you update an existing dashboard
+   Notes can be helpful if you need to revert the dashboard to a previous version. -->
```
Comment (imatwawana) [inline docs/sources/dashboards/build-dashboards/create-dashboard/index.md:57]:
```suggestion
```

File: docs/sources/dashboards/build-dashboards/create-dashboard/index.md [hunk 8]
```diff
@@ -64,28 +66,30 @@ To see an example of repeating rows, refer to [Dashboard with repeating rows](ht
 
 **To configure repeating rows:**
 
-1. On the dashboard home page, click **Add panel**.
+1. Click **Dashboards** in the left-side menu.
+1. Navigate to the dashboard you want to work on.
+1. At the top of the dashboard, click **Add** and select **Row** in the dropdown.
 
-1. On the **Add a panel** dialog box, click **Add a new row**.
+   If the dashboard is empty, you can just click the **+ Add row** button in the middle of the dashboard.
```
Comment (chri2547) [inline docs/sources/dashboards/build-dashboards/create-dashboard/index.md:73]:
```suggestion
   If the dashboard is empty, you can click the **+ Add row** button in the middle of the dashboard.
```

**Review by chri2547 — APPROVED** [review]
left a suggestion. Thank you!

**Pull request merged**